### PR TITLE
Drop pkglibexecdir and use native CMake file configuration

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -31,5 +31,5 @@ target_link_libraries(${SERVICE_EXEC}
 install(
     TARGETS
         ${SERVICE_EXEC}
-    RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_PKGLIBEXECDIR}
+    RUNTIME DESTINATION "${CMAKE_INSTALL_FULL_LIBEXECDIR}/${CMAKE_PROJECT_NAME}"}
```
Comment (sunweaver) [inline src/CMakeLists.txt:34]: Typo at end-of-line...

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please fix the typo (cause of FTBFS on Travis CI).

File: data/ayatana-indicator-display.conf.in [hunk 2]
```diff
@@ -6,4 +6,4 @@ stop on desktop-end or ayatana-indicator-services-end
 respawn
 respawn limit 2 10
 
-exec @pkglibexecdir@/ayatana-indicator-display-service
+exec @CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-display/ayatana-indicator-display-service
```
Comment (sunweaver) [inline data/ayatana-indicator-display.conf.in:9]: use @CMAKE_PROJECT_NAME@ here?

File: src/CMakeLists.txt (hunk 1 above)
Comment (tari01) [reply src/CMakeLists.txt]: Yep, noticed. Fixed now.

File: data/ayatana-indicator-display.service.in [hunk 3]
```diff
@@ -4,7 +4,7 @@ PartOf=graphical-session.target
 PartOf=ayatana-indicators.target
 
 [Service]
-ExecStart=@pkglibexecdir@/ayatana-indicator-display-service
+ExecStart=@CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-display/ayatana-indicator-display-service
```
Comment (sunweaver) [inline data/ayatana-indicator-display.service.in:7]: use @CMAKE_PROJECT_NAME@ here?

**Review by sunweaver — COMMENTED** [review]
Question about CMAKE_PROJECT_NAME...

**Review by sunweaver — APPROVED** [review]
Ok, then...

**Pull request merged**